src/go/rpk/pkg/cli/group/describe.go

```diff
@@ -84,7 +83,7 @@ type describeRow struct {
 	instanceID    *string
 	clientID      string
 	host          string
-	err           error
+	err           string
```

Comment:
but why?

Reply:
Otherwise this is printed as `<nil>`, which isn't the prettiest.